flink/v1.13/flink/src/main/java/org/apache/iceberg/flink/TypeToFlinkType.java

```diff
@@ -108,10 +108,10 @@ public LogicalType primitive(Type.PrimitiveType primitive) {
         Types.TimestampType timestamp = (Types.TimestampType) primitive;
         if (timestamp.shouldAdjustToUTC()) {
           // MICROS
-          return new LocalZonedTimestampType(6);
+          return new LocalZonedTimestampType(3);
```

Comment:
why this change? same question for line 114.

Reply:
Because watermark of flink just support timestamptype(0-3), so i change this.

Reply:
Iceberg timestamp is micro-second only. I am afraid that we can't do this.

Reply:
Yes, we should not change it here. It can be checked outside.